rest/blip_client_test.go

```diff
@@ -136,6 +148,12 @@ func (btr *BlipTesterReplicator) initHandlers(btc *BlipTesterClient) {
 		revID := msg.Properties[revMessageRev]
 		deltaSrc := msg.Properties[revMessageDeltaSrc]
 
+		if btr.forcedErrors.CloseConnectionAtRev != 0 && msg.SerialNumber() == btr.forcedErrors.CloseConnectionAtRev {
+			base.Warnf(base.KeyAll, "Test: Forcing CloseConnectionAtRev error for msg %s", msg)
+			btr.bt.sender.Close()
```

Comment:
Can this modify the BlipTesterReplicator in a way that tests can evaluate (i.e. replication stopped/closed/terminated)?